common/src/constants.rs

```diff
@@ -0,0 +1,4 @@
+// Consensus constants
+
+// https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/p2p-interface.md#configuration
+pub const MAX_REQUEST_LIGHT_CLIENT_UPDATES: u8 = 128;
```

Comment:
Do you anticipate this constant needing to be exposed to any other package other than consensus? If not, can we move the constants module into the consensus package?

Other than that looks good